src/main/java/com/example/demo/services/lifebalancechecker/strategy/LifeBalancerStrategy.java

```diff
@@ -0,0 +1,7 @@
+package com.example.demo.services.lifebalancechecker.strategy;
+
+import com.example.demo.model.LifeBalance;
+
+public interface LifeBalancerStrategy {
+    LifeBalance balanceLife(double desiredSelfCareRatio);
```

Comment:
nie wiem, czy metody w interfejsie do serwisu i strategii o tej samej sygnaturze to ok. dla mnie to mylące, z czym mam do czynienia. ale jak to dobra praktyka, to ok. poproszę źródła :P